saskatoon/saskatoon/settings.py

```diff
@@ -232,10 +256,14 @@ def get_drf_template_mode() -> str:
     'DEFAULT_PAGINATION_CLASS': 'saskatoon.pagination.BasicPageNumberPagination',
 }
 
+# Redis configuration - use Docker service name when in Docker, localhost for local dev
+REDIS_HOST = os.getenv('SASKATOON_REDIS_HOST', '127.0.0.1')
+REDIS_PORT = os.getenv('SASKATOON_REDIS_PORT', '6379')
```

Comment:
can you please add these two entries to the `.env` template ?

Reply:
Agreed! Good idea